_docs/configure/index.md

```diff
@@ -1,5 +1,48 @@
 ---
-title: Configure 
+title: Forseti
 ---
 
-TODO
\ No newline at end of file
+# {{ page.title }}
+
+This page describes how to configure Forseti server once it's set up. Forseti configurations
```

Comment:
I don't think we refer to "Forseti server" anywhere else. Can we just say "Forseti"?

Reply:
Updated.